Modules/_threadmodule.c

```diff
@@ -1595,41 +1595,47 @@ PyInit__thread(void)
     timeout_max = floor(timeout_max);
 
     v = PyFloat_FromDouble(timeout_max);
-    if (!v)
-        return NULL;
-    if (PyModule_AddObject(m, "TIMEOUT_MAX", v) < 0)
-        return NULL;
+    if (PyModule_Add(m, "TIMEOUT_MAX", v) < 0)
```

Comment:
Maybe avoid the temporary variable:

```suggestion
    if (PyModule_Add(m, "TIMEOUT_MAX", PyFloat_FromDouble(timeout_max)) < 0)
```